django_ipam/base/admin.py

```diff
@@ -47,11 +53,46 @@ def change_view(self, request, object_id, form_url="", extra_context=None):
             extra_context = {'labels': labels,
                              'values': values,
                              'used_ip': used_ip,
+                             'subnet': instance,
                              'ipaddress_add_url': ipaddress_add_url,
                              'ipaddress_change_url': ipaddress_change_url}
 
         return super(AbstractSubnetAdmin, self).change_view(request, object_id, form_url, extra_context)
 
+    def get_urls(self):
+        urls = super().get_urls()
+        custom_urls = [
+            re_path(r'^(?P<subnet_id>[^/]+)/export-ipaddress/',
+                    self.export_ipaddress,
+                    name="export_ipaddress"),
+            path('import/', self.import_ipaddress, name="import_ipaddress"),
+        ]
+        return custom_urls + urls
+
+    def export_ipaddress(self, request, subnet_id):
+        response = HttpResponse(content_type='text/csv')
+        response['Content-Disposition'] = 'attachment; filename="ip_address.csv"'
+        writer = csv.writer(response)
+        Subnet.export_csv(self, subnet_id, writer)
+        return response
+
+    def import_ipaddress(self, request):
+        form = IpAddressImportForm()
+        form_template = 'admin/django-ipam/subnet/import.html'
+        if request.method == 'POST':
+            form = IpAddressImportForm(request.POST, request.FILES)
+            if form.is_valid():
+                file = request.FILES['csvfile']
+                if not file.name.endswith(('.csv', '.xls', '.xlsx')):
+                    messages.error(request, 'File type not supported.')
```

Comment:
strings that are to be shown to users should always be `i18n` (internationalization) ready, eg: `_('File type not supported.')`